components/producers/dependency-check/examples/requirements.txt

```diff
@@ -0,0 +1,20 @@
+Django==1.11.29
```

Comment:
just groked that this is used as example vulnerable input can you please add a 
`\# comment at the top saying that this is example input data to dependency-track`
? also would be good to add a couple known vulnerable pakcages so we know what to expect